README.md

```diff
@@ -283,6 +293,12 @@ If you don't have Holesky Eth, these are some useful faucets:
 
 #### Using Docker
 
+Ensure you have the following installed:
+- [Docker](https://docs.docker.com/get-docker/)
+- [Docker Compose](https://docs.docker.com/compose/install/
```

Comment:
) missing

Reply:
Solved in [57c8264](https://github.com/yetanotherco/aligned_layer/pull/419/commits/57c826441db89be099c8f26c99d12cbda43e1c45).